nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/RESTManagementFeature.java

```diff
@@ -31,5 +31,5 @@
 @Deploy("org.nuxeo.ecm.platform.web.common")
 @Deploy("org.nuxeo.rest.management")
 @Deploy("org.nuxeo.rest.management:OSGI-INF/test-webengine-servletcontainer-contrib.xml")
-public class ManagementFeature implements RunnerFeature {
+public class RESTManagementFeature implements RunnerFeature {
```

Comment:
We're used to write _Rest_, can you update it to be consistent with platform? Can you also rename the base test class to follow its feature?